### PR TITLE
DOC-716 | Spring Data and Spring Boot Starter tutorials

### PR DESCRIPTION
**Pull request by Simran-B**
#### Upstream PRs

- 3.10: 
- 3.11: 
- 3.12: 
- 3.13:

### CODE REVIEW

File: site/content/3.11/develop/integrations/_index.md [hunk 1]
```diff
@@ -13,12 +13,22 @@ low-level network communication.
...
+- [Tutorial](spring-data-arangodb/_index.md#get-started)
 - Repository: [github.com/arangodb/spring-data](https://github.com/arangodb/spring-data)
-- [Changelog](https://github.com/arangodb/spring-data/blob/master/ChangeLog.md#readme)
+- [Changelog](https://github.com/arangodb/spring-data/blob/main/ChangeLog.md)
+
+## Spring Boot Starter
+
+The [**Spring Boot Starter**](spring-boot-arangodb.md) for ArangoDB is a set of
+convenient dependency descriptors that you can include in your application based
+on the Spring framework.
```
Comment (Simran-B) [inline site/content/3.11/develop/integrations/_index.md:27]: I'm not sure how helpful this description is for someone who hasn't used Spring

File: site/content/3.11/develop/integrations/spring-boot-arangodb.md [hunk 2]
```diff
@@ -25,39 +28,1476 @@ rules, i.e. minor or patch releases may introduce new features or breaking
...
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.arangodb</groupId>
+            <artifactId>arangodb-spring-boot-starter</artifactId>
+            <version>3.1-0</version>
```
Comment (Simran-B) [inline site/content/3.11/develop/integrations/spring-boot-arangodb.md:78]: Should we add comments here that users should check what versions are available and use the latest suitable version?

File: site/content/3.11/develop/integrations/spring-data-arangodb/_index.md [hunk 3]
```diff
@@ -37,66 +39,131 @@ Spring Data ArangoDB is compatible with:
...
 
-```xml
-<dependency>
-  <groupId>com.arangodb</groupId>
-  <artifactId>arangodb-spring-data</artifactId>
-  <version>x.x.x</version>
-</dependency>
-```
+For a more extensive tutorial about the features of Spring Data ArangoDB and
+Spring Boot support, see the [Spring Boot Starter](../spring-boot-arangodb.md)
```
Comment (Simran-B) [inline site/content/3.11/develop/integrations/spring-data-arangodb/_index.md:48]: I added "and Spring Boot support" in order to remove the separate headline with the paragraph that said "Spring Boot support is offered by Spring Boot Starter ArangoDB" but doesn't this demo use Spring Boot? Doesn't seem quite right

File: site/content/3.11/develop/integrations/spring-data-arangodb/_index.md [hunk 4]
```diff
@@ -37,66 +39,131 @@ Spring Data ArangoDB is compatible with:
...
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.arangodb</groupId>
+            <artifactId>arangodb-spring-data</artifactId>
+            <version>4.4.2</version> <!-- Substitute with the latest Spring Data ArangoDB version -->
```
Comment (Simran-B) [inline site/content/3.11/develop/integrations/spring-data-arangodb/_index.md:92]: If we say to use the latest version here, I suppose we should also do that above for the other deps?

File: site/content/3.11/develop/integrations/spring-data-arangodb/_index.md [hunk 5]
```diff
@@ -184,22 +250,74 @@ public ArangoDB.Builder arango() {
...
+```shell
+mvn spring-boot:run
+```
+
+This should produce a console output similar to:
+
+```
+Ned Stark saved in the database: Character[id=2029, name=Ned, surname=Stark]
+```
+
```
Comment (Simran-B) [inline site/content/3.11/develop/integrations/spring-data-arangodb/_index.md:313]: Is this equally applicable to v3 and v4?

File: site/content/3.11/develop/integrations/_index.md [hunk 6]
```diff
@@ -13,12 +13,22 @@ low-level network communication.
...
+The [**Spring Data integration**](spring-data-arangodb/_index.md) for ArangoDB
+lets you use ArangoDB as a database system in Spring-based Java applications.
 
-- Online course: [Spring Data Tutorial](https://university.arangodb.com/courses/spring-data-tutorial)
+- [Tutorial](spring-data-arangodb/_index.md#get-started)
 - Repository: [github.com/arangodb/spring-data](https://github.com/arangodb/spring-data)
-- [Changelog](https://github.com/arangodb/spring-data/blob/master/ChangeLog.md#readme)
+- [Changelog](https://github.com/arangodb/spring-data/blob/main/ChangeLog.md)
+
+## Spring Boot Starter
```
Comment (Simran-B) [inline site/content/3.11/develop/integrations/_index.md:23]: Does it make sense to list it separately from Spring Data?

File: site/content/3.11/develop/integrations/_index.md (hunk 1 above)
Comment (rashtao) [reply site/content/3.11/develop/integrations/_index.md:27]: Feel free to change it.

File: site/content/3.11/develop/integrations/spring-data-arangodb/_index.md (hunk 4 above)
Comment (rashtao) [reply site/content/3.11/develop/integrations/spring-data-arangodb/_index.md:92]: Yes, but note that:
- Spring Data ArangoDB 4.*.* is compatible with all the still supported Spring Boot 3.*.* (see https://docs.arangodb.com/stable/develop/integrations/spring-data-arangodb/#supported-versions)
- Spring Boot Starter ArangoDB 3.x-* is compatible with Spring Boot 3.x.* (see https://docs.arangodb.com/stable/develop/integrations/spring-boot-arangodb/#supported-versions)

File: site/content/3.11/develop/integrations/spring-data-arangodb/_index.md (hunk 5 above)
Comment (rashtao) [reply site/content/3.11/develop/integrations/spring-data-arangodb/_index.md:313]: Yes, it should be, but v3 EOL now.

**Review by rashtao — COMMENTED** [review]
See comments

**Review by rashtao — APPROVED** [review]

**Pull request merged**